.github/workflows/testing.yml

```diff
@@ -0,0 +1,16 @@
+name: Testing Workflow
+
+on: [push]
+
+permissions:
+  checks: write
+  actions: read
+  contents: read
+
+jobs:
+  lint:
+    name: Lint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: articulate/actions-markdownlint@v1
```

Comment:
Thanks for this. The only comment would whichever is better maintained.

Reply:
I switched to the same one you are using `DavidAnson/markdownlint-cli2-action`.  It does seem to be better since it runs faster because it does not use docker to build an image.